components/match2/wikis/wildrift/match_group_input_custom.lua

```diff
@@ -297,7 +309,7 @@ function matchFunctions.readDate(matchArgs)
 end
 
 function matchFunctions.getTournamentVars(match)
-	match.mode = Logic.emptyOr(match.mode, Variables.varDefault('tournament_mode', 'team'))
+	match.mode = Logic.emptyOr(match.mode, Variables.varDefault('tournament_mode', _TEAM_OPPONENT_TYPE))
```

Comment:
Add a new const `_DEFAULT_MODE = 'team'` and use that here. The _TEAM_OPPONENT_TYPE just happen to have the same value as the default mode, but they are very different